README.md

```diff
@@ -1 +1,62 @@
-# SoFiAX_services
\ No newline at end of file
+## SofiAX_services
+
+This repository provides dode for the deployment of database portals using *docker* containers.
+
+**TOC**
+<HR>
+
++ [Create an environment file](#create-an-environment-file)
++ [Raise the services](#raise-the-services)
++ [Prepare the services](#prepare-the-services)
++ [Access to logs](#access-to-logs)
+
+
+#### Create an environment file
+Create (or modify) ``.env`` file and include the following values:
+
+```
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG=1 
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY=<secret key by generator>
+DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]
+SQL_ENGINE=django.db.backends.postgresql
```

Comment:
I prefer to use `DATABASE_ENGINE` rather than `SQL_ENGINE` as the environment variable name. Same applies to other database configuration env vars.

Reply:
Updated in the PR